docs/datascience/notebooks-web.md

```diff
@@ -5,69 +5,85 @@ TOCTitle: Jupyter Notebooks on the web
 
-### Remote tunneling with VS Code Server
+## Remote tunneling with VS Code Server
 
-You can securely connect to a remote machine without the requirement of SSH by installing the [VS Code Server](https://code.visualstudio.com/docs/remote/vscode-server) on your remote machine and accessing it through VS Code for the Web (you can also connect to the server on VS Code Desktop). To do so:
+You can securely connect to a remote machine without the requirement of SSH by installing the [VS Code Server](/docs/remote/vscode-server.md) on your remote machine and accessing it through VS Code for the Web (you can also connect to the server on VS Code Desktop). To do so:
 
-1. Install the `code` [CLI](https://code.visualstudio.com/#alt-downloads) on your remote machine.
-   > **Note**: You can skip this step if VS Code Desktop is already installed on the remote machine, because the `code` CLI is built into VS Code Desktop.
-2. Create a [secure tunnel](https://code.visualstudio.com/docs/remote/tunnels) with the tunnel command: `code tunnel` (you can also run `code-insiders tunnel` if you wish to use the daily release of VS Code) and follow the prompts to grant access to the server. This will download and start the VS Code Server on your remote machine and then create a tunnel to it.
-3. Use the CLI's output vscode.dev URL tied to this remote machine, such as https://vscode.dev/tunnel/<machine_name>/<folder_name> for access to this remote machine on any client.
-4. Open a notebook file and select any Jupyter kernel or Python environment to run your code
+1. Install the `code` [CLI](/download) on your remote machine.
 
-### Connect to a remote Jupyter server
+   > **Note**: You can skip this step if VS Code Desktop is already installed on the remote machine as the `code` CLI is built into VS Code Desktop.
 
-You can also connect to a remote Jupyter server directly by selecting the **Jupyter: Specify Jupyter Server for Connections** command -> pasting the URL for your remote Jupyter server, e.g., `http://<ip-address>:<port>/?token=<token>`.
+2. Create a [secure tunnel](/docs/remote/tunnels.md) with the tunnel command: `code tunnel` (you can also run `code-insiders tunnel` if you wish to use the [daily release](/insiders) of VS Code) and follow the prompts to grant access to the server. This will download and start the VS Code Server on your remote machine and then create a tunnel to it.
+3. Use the CLI's output vscode.dev URL tied to this remote machine, such as `https://vscode.dev/tunnel/<machine_name>/<folder_name>` for access to this remote machine on any client.
+4. Open a notebook file and select any Jupyter kernel or Python environment to run your code.
+
+## Connect to a remote Jupyter server
+
+You can also connect to a remote Jupyter server directly by selecting the **Jupyter: Specify Jupyter Server for Connections** command and then pasting the URL for your remote Jupyter server.
```

Comment:
You can also connect to any remote Jupyter server by pasting the URL with the format: `http://<ip-address>:<port>/?token=<token>`.

To do so, selecting the **Jupyter: Specify Jupyter Server for Connections** command and then pasting the URL for your remote Jupyter server.

![Enter Jupyter server](images/notebooks-web/select-enter-server-url.png)

To enter your remote Jupyter server URL with older versions of VS Code and Jupyter extension:
1. Click **Jupyter Server: ...** on the status bar
![Jupyter Server status bar]()
2. Select **Existing**
![Select existing Jupyter server]()

Reply:
For "Jupyter server status bar":
![jupyter-server-status-bar](https://user-images.githubusercontent.com/10833267/213820308-de383dea-01bf-4707-a434-f1d4e2a1bbf6.png)

For "Select existing Jupyter server":
![select-existing-old](https://user-images.githubusercontent.com/10833267/213820257-bbcd711a-19c1-4c71-975a-2857799fa645.png)